### DISCUSSION TITLE
3D Secure doesn't show purchase amount when tokenizing payment

### DISCUSSION THREAD

**Discussion by Buupu** [swellstores/community#81] [Help] [answered]
When authenticating cards the value of the purchase is displaying 0.00 when I'd expect to see the cart total.

I've tested this with amex/visa/mastercard and they all have the same issue - 

<img width="429" alt="image" src="https://user-images.githubusercontent.com/26443341/211815674-e535ffff-dcc3-496b-81d4-52b0c16d2386.png">

Is this intended behaviour? 

I feel like it would appear misleading to the customer.

Any help would be appreciated.

**Comment by swellmike** [comment]
This sounds like a bug.  Will forward it to the Swell team.

**Comment by jesseswell** [comment]
Hey @Buupu!

I checked your store and just to confirm, this is for the Stripe Gateway correct and using the Swell-hosted checkout?

**Reply by Buupu** [reply]
Hey @jesseswell thanks for looking at this.

It's for the Stripe gateway using the swell-js implemention.

The 3DS triggers when the `swell.payment.tokenize(...)` is called.

**Comment by jesseswell** [comment]
@Buupu thank you for the additional information. I'll relay this to our team.

**Comment by jesseswell** [comment]
@Buupu can you share what you're seeing in Stripe for the payment intent?

**Reply by Buupu** [reply]
Sure, here you go - 

```
{
    "id": "pi_3MPjYeDckEU4DM7z0GtpcQhr",
    "client_secret": "pi_3MPjYeDckEU4DM7z0GtpcQhr_secret_GynUa8aOUrJ6EB794BvvoOB4l",
    "amount": 100,
    "status": "requires_capture",
    "currency": "gbp"
}
```

**Reply by Buupu** [reply]
Hello, is there an ETA on when this might be resolved?

**Comment by swellmike** [comment, marked as answer]
We've deployed a fix for this to our hosted Checkout.  Please let us know if this works for your use case.

**Reply by Buupu** [reply]
Thank you for getting a fix up!

I think you still need to publish the release though.

Thanks in advance

**Reply by swellmike** [reply]
The swell-js SDK should be updated now as well.  v3.19.4

**Reply by Buupu** [reply]
All fixed! Thanks again
